firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Containers/EnumMap.txt

```diff
@@ -0,0 +1,357 @@
+SCENARIO('The EnumMap is constructed from initializer list works corretly') 
+  GIVEN('An EnumMap constructed with capacity 11, has {1, 1}, {2,2}, {3,3}, {4,4} items') 
```

Comment:
It would be clearer to specify the Key and Value types used for specifying the EnumMap template.